app/services/authn_context_resolver.rb

```diff
@@ -1,39 +1,63 @@
 # frozen_string_literal: true
 
 class AuthnContextResolver
-  attr_reader :service_provider, :vtr, :acr_values
+  attr_reader :user, :service_provider, :vtr, :acr_values
 
-  def initialize(service_provider:, vtr:, acr_values:)
+  def initialize(user:, service_provider:, vtr:, acr_values:)
+    @user = user
     @service_provider = service_provider
     @vtr = vtr
     @acr_values = acr_values
   end
 
   def resolve
     if vtr.present?
-      vot_parser_result
+      selected_vtr_parser_result_from_vtr_list
     else
       acr_result_with_sp_defaults
     end
   end
 
   private
 
-  def vot_parser_result
-    @vot_result = Vot::Parser.new(
-      vector_of_trust: vtr&.first,
-      acr_values: acr_values,
-    ).parse
+  def selected_vtr_parser_result_from_vtr_list
+    if biometric_proofing_vot.present? && user&.identity_verified_with_biometric_comparison?
+      biometric_proofing_vot
+    elsif non_biometric_identity_proofing_vot.present? && user&.identity_verified?
+      non_biometric_identity_proofing_vot
+    else
+      parsed_vectors_of_trust.first
```

Comment:
I am testing this in combination with https://github.com/18F/identity-oidc-sinatra/pull/163 and it looks like this is always picking `C1.P1` from the vtr `["C1.P1","C1"]`. Is this correct? If we are trying to replicate ialmax, I would expect `C1` to be used in those cases (this is assuming I understand what ialmax is supposed to do)

Reply:
Wait, reading a little more I think i am confused

Reply:
You are right, there is a little bug in here. Let me fix it up. I tested with `C1,C1.P1` when I should be testing with `C1.P1,C1`.

Reply:
https://github.com/18F/identity-idp/pull/10517/commits/476fc5a64ca8b5deef46cc225446bee7a7d33044 has the fix